public/index.html

```diff
@@ -1,10 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-88778470-5"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-88778470-5');
+    </script>
```

Comment:
Do they say they want this at the top of the head now?

Let's move this to the very last thing in the head if not. I don't want everything else in the site getting hung up by this script tag getting executed.

Reply:
yup, they do. it's an `async` script so it doesn't hang the page

Reply:
I guess they want to make sure they're the first network call so they can see other network metrics? Kinda annoying lol. They've gotten more aggressive with its placement over the tears.

Reply:
years*, but also tears